leo/commands/clean.rs

```diff
@@ -14,45 +14,39 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{cli::*, cli_types::*, errors::CLIError};
-use leo_package::{
-    outputs::{ChecksumFile, ProofFile, ProvingKeyFile, VerificationKeyFile},
-    root::Manifest,
-};
-
-use clap::ArgMatches;
+use crate::{commands::Command, context::Context};
+use anyhow::Result;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates